DARPA-ASKEM/TERArium
# feat: fetch and display model metadata and parameters

![image](https://user-images.githubusercontent.com/16928557/203416481-be8181c7-33ec-483b-961a-4282ed787b75.png)


# Description

Fetch a model's name, description, and parameters and display them in the right side panel on the model's page.

This is only the first pass. Remaining work:
- Add axios and replace `uncloak()`
- Currently we always fetch the model with ID `1`. We'll want to change that to use the selected model ID, presumably from the URL
- Whole bunch of styling clean up: https://github.com/DARPA-ASKEM/TERArium/issues/169
- The petri net that's displayed currently is hardcoded, unrelated to the name and metadata we now display: https://github.com/DARPA-ASKEM/TERArium/issues/165

Resolves #160 and #162

## Type of change


# How Has This Been Tested?

To test:

1. Set up the orchestration repo (follow the steps in CONTRIBUTING.md and let me know if you run into issues)
2. Run `./deploy-terarium dev` in the `kubernetes/local` folder in the orchestration repo to run the gateway
3. Set up and run the data-service (`make up` then `make populate`) to give us an API with data to access
4. Go into `application.properties` in this repository and uncomment line 26 to use 8001 as the port for the data service
5. From `packages/services/hmi-server` run `quarkus dev` to launch the hmi-server
6. From the root of this repository, run `yarn workspace hmi-client run dev` to run the frontend
7. Navigate to `localhost:8078` and sign in
8. Navigate to `http://localhost:8078/app/#/projects/1/model` to see this model page

You should see a model name above the graph and model metadata down the right side, as in the screenshot above.

# Checklist:
